crates/kitsune_p2p/dht/src/spacetime/topology.rs

```diff
@@ -330,7 +313,7 @@ pub struct GossipParams {
     /// a time buffer of 2 will allow +/- 10 min discrepancies with gossip partners.
     pub max_time_offset: TimeQuantum,
 
-    /// What difference in power will you accept for other agents' Arqs?
+    /// What is the difference in power will you accept for other agents' Arqs?
```

Comment:
the old sentence actually made more sense

```suggestion
    /// What is the difference in power that you will accept in other agents' Arqs?
```